packages/components/src/confirm/index.ts

```diff
@@ -0,0 +1,15 @@
+//@ts-nocheck
```

Comment:
Just noting that this comment should eventually be removed

Reply:
Yes 👍🏻  I'll take care of this before promoting this PR.